src/Symfony/Component/Translation/Loader/YamlFileLoader.php

```diff
@@ -54,7 +54,7 @@ public function load($resource, $locale, $domain = 'messages')
         try {
             $messages = $this->yamlParser->parse(file_get_contents($resource));
         } catch (ParseException $e) {
-            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s"', $resource), 0, $e);
+            throw new InvalidResourceException(sprintf('Error parsing YAML, invalid file "%s" at line %d', $resource, $e->getParsedLine()), 0, $e);
```

Comment:
i think we should not replicate logic from ParseException, just use it's message.